packages/ui-extensions/src/surfaces/checkout/shared.ts

```diff
@@ -1,71 +1,35 @@
+/**
+ * The list of supported components.
+ * As of April 15th, 2025, this is a subset of the components that will be available in the 2025-07 stable release.
+ */
```

Comment:
These comments are outdated

Reply:
I think this should be changed on checkout-web then.